csrc/quantization/gptq/compat.cuh

```diff
@@ -0,0 +1,56 @@
+#ifndef _compat_cuh
```

Comment:
For all the CPP kernel files, can you comment where the code is originally from if it's from other projects?